test/Ruby/ParseSpec.hs

```diff
@@ -51,9 +51,18 @@ stringParseSpec =
 rubyStringArray :: Text
 rubyStringArray = "[ \"hello\",\"world\" ,\t\"foo\",\"bar\"]"
 
+commentedRubyStringArray :: Text
+commentedRubyStringArray = "[ #cmt\n\"hello\"#cmt\n,\"world\" ,\t\"foo\",\"bar\"]"
```

Comment:
Please prefer string literals. Example: https://github.com/fossas/fossa-cli/blob/3251a54302faccacf7a79cce09eb8d861e0faf0e/test/App/Fossa/VSI/DynLinked/Internal/Lookup/RPMSpec.hs#L32-L59

Only commenting here, but applies generally.

Reply:
Oh thank you, I tripped over myself a few times writing these tests. I will update this and hopefully it will be a bit easier to understand.